client/lib/preferences-helper/PreferenceList.js

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { partial, isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import { getAllRemotePreferences } from 'state/preferences/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+class PreferenceList extends Component {
+	render() {
+		const { preferences, translate, unsetPreference } = this.props;
+		return (
+			<div>
+				<a
+					href={ '/devdocs/client/state/preferences/README.md' }
+					title={ translate( 'Preferences' ) }
+				>
+					{ translate( 'Preferences' ) }
+				</a>
+				<Card className="preferences-helper__current-preferences">
+					{ ! isEmpty( preferences ) ? (
+						Object.keys( preferences ).map( prefName => (
+							<div key={ prefName }>
+								<h5 className="preferences-helper__preference-header">{ prefName }</h5>
+								<span className="preferences-helper__value-label">
+									{ translate( 'Value' ) + ':' }
+								</span>
+								<span className="preferences-helper__value">
+									{ preferences[ prefName ].toString() }
+								</span>
+								<button
+									className="preferences-helper__unset"
+									onClick={ partial( unsetPreference, prefName, null ) }
+								>
+									{ translate( 'unset' ) }
+								</button>
+							</div>
+						) )
+					) : (
+						<h5 className="preferences-helper__preference-header">
+							{ translate( 'No Preferences' ) }
+						</h5>
+					) }
+				</Card>
+			</div>
+		);
+	}
+}
+
+export default connect(
+	state => {
+		const preferences = getAllRemotePreferences( state );
+		return {
+			preferences,
+		};
```

Comment:
Super minor but we could remove the `preferences` variable since it's not really necessary here, and use a shorter syntax:

```javascript
state => ( {
    preferences: getAllRemotePreferences( state )
} ),
```